modules/post/multi/manage/shell_to_meterpreter.rb

```diff
@@ -249,24 +264,34 @@ def transmit_payload(exe, platform)
       #
       sent = 0
       aborted = false
-      cmds.each do |cmd|
-        ret = cmd_exec(cmd)
-        if !ret
-          aborted = true
-        else
-          ret.strip!
-          aborted = true if !ret.empty? && ret !~ /The process tried to write to a nonexistent pipe./
-        end
-        if aborted
-          print_error('Error: Unable to execute the following command: ' + cmd.inspect)
-          print_error('Output: ' + ret.inspect) if ret && !ret.empty?
-          break
-        end
 
-        sent += cmd.length
+      if cmds.length >= 2
+        cmds.take(cmds.length - 1).each do |cmd|
+
+          # These commands need to be run in order.
+          ret = cmd_exec(cmd, nil, datastore['Timeout'] || 15, { 'Channelized' => true })
```

Comment:
It looks like `Timeout` isn't an option in this module by default unless I'm missing something  👀 

Potentially we should add a new option for `CommandTimeout` to make it clear that each command execution will individually have its own timeout applied - rather than being some sort of global `Timeout`